Legiones Astartes.cat

```diff
@@ -11852,34 +11852,36 @@ If at any point there are only Rapier Carrier Models remaining in this Unit, tho
-                  </costs>
-                </infoLink>
-                <infoLink name="Heavy bolter" id="98ef-7a9d-a40c-b44b" hidden="false" type="selectionEntry" targetId="bc40-b14a-978e-0232" defaultAmount="2" sortIndex="3">
-                  <costs>
-                    <cost name="Point(s)" typeId="9893-c379-920b-8982" value="0"/>
-                  </costs>
-                </infoLink>
-                <infoLink name="Volkite culverin" id="9142-45c5-f92b-83a1" hidden="false" type="selectionEntry" targetId="fcf0-9c1b-d51e-6bc7" sortIndex="5">
-                  <costs>
-                    <cost name="Point(s)" typeId="9893-c379-920b-8982" value="5"/>
-                  </costs>
-                </infoLink>
-                <infoLink name="Cyclone missile launcher" id="8a7a-cd98-a6e2-8b3e" hidden="false" type="selectionEntry" targetId="b341-159d-9dfb-ff31" sortIndex="1">
-                  <costs>
-                    <cost name="Point(s)" typeId="9893-c379-920b-8982" value="0"/>
-                  </costs>
-                </infoLink>
-              </infoLinks>
+            <selectionEntryGroup name="May exchange cyclone missile launcher for:" id="fc82-2177-9a0c-2010" hidden="false" sortIndex="1" defaultSelectionEntryId="c357-41d9-a1b2-2104">
+              <entryLinks>
+                <entryLink import="true" name="Cyclone missile launcher" hidden="false" id="c357-41d9-a1b2-2104" type="selectionEntry" targetId="b341-159d-9dfb-ff31" sortIndex="1" defaultAmount="1"/>
+                <entryLink import="true" name="Heavy flamer" hidden="false" id="00ab-2091-0394-0825" type="selectionEntry" targetId="4fa7-ea45-2a89-ad86" sortIndex="3">
+                  <modifiers>
+                    <modifier type="set" value="Two Heavy Flamers" field="name"/>
+                  </modifiers>
+                </entryLink>
+                <entryLink import="true" name="Lascannon" hidden="false" id="e05d-b780-1c55-3bb2" type="selectionEntry" targetId="b6d5-4be2-c37e-2197" sortIndex="4">
+                  <modifiers>
+                    <modifier type="set" value="5" field="9893-c379-920b-8982"/>
+                    <modifier type="set" value="Two Lascannon" field="name"/>
+                  </modifiers>
+                </entryLink>
+                <entryLink import="true" name="Heavy bolter" hidden="false" id="a096-7d95-304d-985d" type="selectionEntry" targetId="bc40-b14a-978e-0232" sortIndex="2">
+                  <modifiers>
+                    <modifier type="set" value="Two Heavy Bolters" field="name"/>
+                  </modifiers>
+                </entryLink>
+                <entryLink import="true" name="Volkite culverin" hidden="false" id="cf9e-375b-8561-ac8c" type="selectionEntry" targetId="fcf0-9c1b-d51e-6bc7" sortIndex="5">
+                  <modifiers>
+                    <modifier type="set" value="5" field="9893-c379-920b-8982"/>
+                    <modifier type="set" value="Two Volkite culverin" field="name"/>
```

Comment:
This requsted change still wasn't fixed, unless I just failed to cherry-pick the fixing commit